server/tserver/src/main/java/org/apache/accumulo/tserver/compaction/TwoTierCompactionStrategy.java

```diff
@@ -61,6 +64,18 @@ public void verifyRequiredProperties(Object... objectsToVerify) throws IllegalAr
     }
   }
 
+  /**
+   * Calculates the total size of input files in the compaction plan
+   */
+  private Long calculateTotalSize(MajorCompactionRequest request, CompactionPlan plan) {
+    long totalSize = 0;
+    Map<FileRef,DataFileValue> allFiles = request.getFiles();
+    for (FileRef fileRef : plan.inputFiles) {
```

Comment:
They should be at this point since this class extends DefaultCompactionStrategy and calls super.getCompactionPlan() to get the plan. If you look at DefaultCompactionStrategy.findMapFilesToCompact() it creates the plan.inputFiles from the request...
`    for (Entry<FileRef,DataFileValue> entry : request.getFiles().entrySet()) {
      candidateFiles.add(new CompactionFile(entry.getKey(), entry.getValue().getSize()));
    }`